lib/source-loader/extract-source.js

```diff
@@ -1 +0,0 @@
-module.exports = require('./dist/cjs/extract-source');
```

Comment:
Can we leave this in and just not use it, and add a deprecation warning? I'm worried about breaking semver.

Reply:
I added it back, with a deprecation warning.